rasa/cli/data.py

```diff
@@ -498,5 +539,23 @@ def _get_rules_path(path: Text) -> Path:
         rasa.shared.utils.cli.print_error_and_exit(
             f"'{rules_file}' needs to be the path to a file."
         )
+    elif rules_file.is_file():
+        rasa.shared.utils.cli.print_info(
+            f"Output file '{rules_file}' did not exist and will be created."
+        )
+        rasa.shared.utils.io.create_directory_for_file(rules_file)
```

Comment:
the dir must already exist `is_file()` is true

Reply:
I guess the check should be:
```suggestion
    if not rules_file.parent.is_dir():
        rasa.shared.utils.cli.print_info(
            f"Output file '{rules_file}' did not exist and will be created."
        )
        rasa.shared.utils.io.create_directory_for_file(rules_file)
```

Reply:
`is_file` also checks if the file (and hence the parent dir) exists. And the directory mustn't exist as we create it with `create_directory_for_file`

Reply:
yes that's right, but you now have an 
```
elif rules_file.is_file()
```
which means the dir would only be created if the file (and hence the parent dir) already exists

Reply:
oh dear. of course 🙈